src/frontend/main/MainWrapper.tsx

```diff
@@ -65,14 +65,18 @@ const Container = styled.div`
   margin: 0 auto;
 `;
 
-const TotalResponses = styled.div`
+const TotalResponses = styled(Container)<{ topBorder?: boolean }>`
+  border-top: ${({ topBorder, theme }) => (topBorder ? `1px solid ${theme.black}` : 'none')};
+  padding: 10px 0;
+`;
+
+const TotalResponsesWrapper = styled.div`
   background: ${props => props.theme.white};
   position: fixed;
   bottom: 0px;
-  padding: 10px 0 10px 16px;
```

Comment:
Moving padding into the child div to prevent any white space able the top border:

![image](https://user-images.githubusercontent.com/6004147/81658845-d117bb80-9441-11ea-8a27-ad32951d594a.png)